soccer/WindowEvaluator.cpp

```diff
@@ -27,8 +28,11 @@ void WindowEvaluator::createConfiguration(Configuration* cfg) {
         new ConfigDouble(cfg, "WindowEvaluator/distScoreCoeff", 0.3);
 }
 
-WindowEvaluator::WindowEvaluator(SystemState* systemState)
-    : system(systemState) {}
+WindowEvaluator::WindowEvaluator(Context* context) : context(context) {
+    if (context == nullptr) {
+        std::cout << "NULL POINTER FOUND!" << std::endl;
```

Comment:
I feel like this could be more descriptive. If this appears multiple places, it might be annoying to debug since you would have to narrow down exactly which one it is

Reply:
Oh god. I let my lazy debug prints in the code. That's embarrassing, it should just be deleted. I'm honestly surprised the text is even somewhat related to the actual error 😂